common/util/user_interaction.cc

```diff
@@ -57,4 +58,20 @@ char ReadCharFromUser(std::istream& input, std::ostream& output,
   }
 }
 
+namespace term {
+// TODO(hzeller): assumption here that basic ANSI codes work on all
+// platforms, but if not, change this with ifdef.
+static constexpr absl::string_view kBoldEscape("\e[1m");
+static constexpr absl::string_view kInverseEscape("\e[7m");
+static constexpr absl::string_view kNormalEscape("\e[0m");
```

Comment:
`\e` is non-standard - use `\033` or `\x1B`

Reply:
Thanks, I didn't know, I guess I have been using `gcc` and `clang` for too long. Fixed, changed to `\033`